### ISSUE TITLE
 Email: ITI Comments on First Public Working Draft of W3C Accessibility Guidelines (WCAG) 3.0 (3 - adjectival rating example)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#445]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0012.html): 
from [PDF](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/att-0012/ITI_Comments_on_WCAG_3.0_First_Public_Working_Draft_20210224_1_.pdf)

* ITI appreciates that the example in 6.2 for an adjectival rating scale clearly states how things should be measured, however it appears to be an almost impossible bar to meet. If there is an expectation to achieve a 3.5 point overall score, based on the example listed, that would translate to the requirement of a 95% passing level on all tests. Further, the table in 6.2 sets the bar too high (95%). We appreciate the Editors Note that the number could be 90%, which may be more reasonable, underscoring the need for the entire scoring mechanism to be much more clearly stated.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
